src/System.Security.Cryptography.OpenSsl/src/System/Security/Cryptography/ECDsaOpenSsl.cs

```diff
@@ -340,12 +341,31 @@ public SupportedAlgorithm(int keySize, int nid)
         }
 
         private static readonly SupportedAlgorithm[] s_supportedAlgorithms =
-            new SupportedAlgorithm[]
+            RemoveAlgorithmsUnsupportedByOs(
+                new SupportedAlgorithm[]
```

Comment:
I don’t want the implementation details of how we filter the list to leak out into the representation of the list.